package.json

```diff
@@ -56,5 +56,8 @@
     "stylelint": "^13.13.1",
     "stylelint-config-carbon": "^1.7.0",
     "ts-jest": "^27.0.7"
+  },
+  "dependencies": {
+    "@carbon/pictograms-react": "^11.20.0"
```

Comment:
I see you installed this at the workspace level, not the service level. Can you uninstall it from here and do:

`npm --workspace services/web-app install @carbon/pictograms-react@latest`

See docs: https://github.com/carbon-design-system/carbon-platform/blob/main/docs/day-to-day-development.md#adding-new-project-dependencies